MC/bin/o2dpg_sim_workflow.py

```diff
@@ -425,11 +425,6 @@ def createRestDigiTask(name, det='ALLSMALLER'):
    ITSTPCMATCHtask['cmd']= 'o2-tpcits-match-workflow ' + getDPL_global_options(bigshm=True, nosmallrate=True) + ' --tpc-track-reader \"tpctracks.root\" --tpc-native-cluster-reader \"--infile tpc-native-clusters.root\"'
    workflow['stages'].append(ITSTPCMATCHtask)
 
-   # this can be combined with TRD digitization if benefical
-   TRDTRAPtask = createTask(name='trdtrap_'+str(tf), needs=[TRDDigitask['name']], tf=tf, cwd=timeframeworkdir, lab=["DIGI"])
-   TRDTRAPtask['cmd'] = 'o2-trd-trap-sim'
-   workflow['stages'].append(TRDTRAPtask)
```

Comment:
For the record, this change is not enough since it's deleting a "named" graph node which is referenced elsewhere.
I will adapt this PR and merge. Thanks in any case!